### PR TITLE
Fix content type for #118

### LINKED ISSUE #118: Json posted as text
When posting any object to trello I get an error stating, 

> invalid value for text

I was able to drill down to the core problem, it's because the content type header is set by default as text for StringContent, and trello doesn't allow json as text.

### PR DESCRIPTION
**Pull request by eliekstein**
Resolves #118

### CODE REVIEW

File: Manatee.Trello.WebApi/WebApiClient.cs [hunk 1]
```diff
@@ -150,7 +150,10 @@ private static HttpContent GetContent(WebApiRestRequest request)
 				return formData;
 			}
 			var body = TrelloConfiguration.Serializer.Serialize(request.Body);
-			return new StringContent(body);
+			var jsonContent = new StringContent(body);
+            jsonContent.Headers.ContentType = new System.Net.Http.Headers.MediaTypeHeaderValue("application/json");
```
Comment (gregsdennis) [inline Manatee.Trello.WebApi/WebApiClient.cs]: Please:

1. Fix formatting (tabs).
2. Add a `using` instead of explicitly listing the namespace.

**Comment by eliekstein** [comment]
Updated.

**Comment by eliekstein** [comment]
@gregsdennis Are you going to merge and publish anytime soon?

**Comment by gregsdennis** [comment]
I'm at work right now.  I'll handle this today.

**Pull request merged**